website/docs/artist_concepts.md

```diff
@@ -24,6 +26,11 @@ A numbered iteration of a given subset. Each version contains at least one [repr
 
 Each published variant can come out of the software in multiple representations. All of them hold exactly the same data, but in different formats. A model, for example, might be saved as `.OBJ`, Alembic, Maya geometry or as all of them, to be ready for pickup in any other applications supporting these formats.
 
+
+#### Naming convention
+
+At this moment names of assets, tasks, subsets or representations can contain only letters, numbers and underscore.
```

Comment:
_This is not feedback, but a potential nice use case for some users._

Regarding this - we've actually exposed "label" in our Project Manager so that you can do something like this:

![afbeelding](https://user-images.githubusercontent.com/2439881/186120329-5598e03c-cbe6-497d-a525-af647cbbbdd1.png)

Which would show as only the human readable label in Launcher and Loader, etc.

Reply:
I've added that information primarily for ftrack, shotgrid and kitsu users where it would require more complex solution. Story for different place :)